dsl_parser/import_resolver/abstract_import_resolver.py

```diff
@@ -57,8 +55,8 @@ def read_import(import_url):
     error_str = 'Import failed: Unable to open import url'
     if import_url.startswith('file:'):
         try:
-            request = urllib2.Request(import_url)
-            with contextlib.closing(urllib2.urlopen(request)) as f:
+            filename = import_url[len('file:'):]
```

Comment:
While this should work, it's also a little ugly not stripping the full file:// off the beginning isn't it?